src/v2/Components/FlashBanner/index.tsx

```diff
@@ -20,6 +20,7 @@ interface FlashBannerProps {
  *
  */
 export const FlashBanner: React.FC<FlashBannerProps> = props => {
+  if (typeof window === "undefined") return null
```

Comment:
I don't think this is necessary any more - will remove.

Reply:
I am now considering leaving it in - Basically `SystemQueryRenderer` accomplishes this automatically, but I wonder if that isn't clear enough & could somehow cause confusion later.

Reply:
Good rule of thumb is to only write code you know you need and to avoid hypotheticals. No need to add here.